### PR TITLE
[12.0] [MIG] product_state

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**
Product State
=============

This module introduces the state field on product template and allows simple product life cycle.

### CODE REVIEW

**Comment by Nikul-Chaudhary** [comment]
@OCA/product-maintainers  could you please review code?

File: product_state/__manifest__.py [hunk 1]
```diff
@@ -2,12 +2,12 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 {
     'name': "Product State",
-    'description': """
+    'summary': """
```
Comment (rousseldenis) [inline product_state/__manifest__.py:5]: @Nikul-Chaudhary Could you use README generator ?

**Review by rousseldenis — APPROVED** [review]

File: product_state/__manifest__.py (hunk 1 above)
Comment (Nikul-Chaudhary) [reply product_state/__manifest__.py:5]: @rousseldenis @OCA-git-bot  is automatically generated.

Comment (rousseldenis) [reply product_state/__manifest__.py:5]: @Nikul-Chaudhary only if you use README fragments

Comment (Nikul-Chaudhary) [reply product_state/__manifest__.py:5]: @rousseldenis Done

**Review by tbaden — COMMENTED** [review]
can you please generate the readme via maintainer tools

**Comment by rousseldenis** [comment]
> can you please generate the readme via maintainer tools

@tbaden Here, @Nikul-Chaudhary wrotes fragments, so, README will be generated depending on them (he could have generated README too, but not mandatory as it will be automatically be done by bot).

**Review by tbaden — APPROVED** [review]
thanks for this info @rousseldenis

**Review by sebalix — APPROVED** [review]
LGTM

**Review by i-vyshnevska — APPROVED** [review]

**Review by guewen — APPROVED** [review]

**Pull request merged**